src/SFA.DAS.AANHub.Application/StagedApprentices/Queries/GetStagedApprenticeResult.cs

```diff
@@ -0,0 +1,36 @@
+﻿using SFA.DAS.AANHub.Domain.Entities;
+
+namespace SFA.DAS.AANHub.Application.StagedApprentices.Queries
+{
+    public class GetStagedApprenticeResult
```

Comment:
GetStagedApprenticeQueryResult

Reply:
renamed in both Application and Tests proj